### DISCUSSION TITLE
using rtsp proxy get source stream,how to put the "Range" header in RTSP SETUP and PLAY messages.

### DISCUSSION THREAD

**Discussion by zodostffaa** [bluenviron/mediamtx#1054] [Questions]
using rtsp proxy get source stream,how to put the "Range" header in RTSP SETUP and PLAY messages. the source stream server is RTSP specs (RFC 2326)

**Comment by aler9** [comment]
Hello, this is currently unsupported as no one asked for it yet, but if it's requested by more people, it will be implemented.

**Comment by throberto** [comment]
it would be interesting to have this

**Comment by aler9** [comment]
moved here #1096
